jenkins-ci/Jenkinsfile

```diff
@@ -0,0 +1,57 @@
+pipeline {
+    agent any
+    stages {
+        stage('all-in-one: clean environment') { 
+            steps { 
+                sh 'oc get all'
+                sh 'oc delete all,template,daemonset -l jaeger-infra'
+                sh 'oc get all'
+            }
+        }
+        stage('all-in-one: deploy Jaeger') { 
+            steps { 
+                sh 'oc get all'
+                sh 'oc process -f all-in-one/jaeger-all-in-one-template.yml | oc create -f -'
+                sh 'oc get all'
+            }
+        }
+        stage('all-in-one: verify deployment'){
+            steps {
+                sh 'oc get dc'
+                openshiftVerifyDeployment apiURL: '', authToken: '', depCfg: 'jaeger', namespace: '', replicaCount: '1', verbose: 'true', verifyReplicaCount: 'true', waitTime: '3', waitUnit: 'min'
```

Comment:
I'd recommend increasing the waitTime here to at least 5 minutes.  I've had this fail a couple of times because of problems downloading images.